src/StakeRegistry.sol

```diff
@@ -386,35 +386,26 @@ contract StakeRegistry is StakeRegistryStorage {
             return;
         }
 
-        uint96 prevStake;
+        // Get our last-recorded stake update
         uint256 historyLength = _totalStakeHistory[quorumNumber].length;
```

Comment:
revert if 0

Reply:
I need to add an explicit comment about this, but this method assumes the quorum exists. all calling locations already perform this check, and will revert higher in the stack if it's caught - so we don't necessarily need an explicit check here.

`updateStakes` doesnt currently check this, but i'm moving that out of this contract anyway.